src/ui/components/atoms/url-input/URLinput.module.scss

```diff
@@ -0,0 +1,371 @@
+@import "../../../../globals/variables";
+
+.container {
+    box-sizing: border-box;
+    color: $ads-text-main-hover;
+    font-family: Poppins, Arial, sans-serif;
+    width: 22rem;
+
+    @media (width <= 600px) {
+        min-width: 0;
+        padding: 1rem;
+        width: 100%;
+    }
+}
+
+.header {
+    align-items: center;
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: .5rem;
+}
+
+.label-wrapper {
+    align-items: center;
+    display: flex;
+    gap: .3rem;
+}
+
+.label-icon {
+    height: 1rem;
+    width: 1rem;
+
+    path {
+        fill: currentcolor;
+    }
+}
+
+.close-button {
+    background: none;
+    border: none;
+    cursor: pointer;
+    padding: 0;
+
+    &:hover {
+        opacity: 0.8;
+    }
+}
+
+.close-icon-url {
+    height: 1rem;
+    width: 1rem;
+
+    path {
+        fill: currentcolor;
+    }
+}
+
+.input-container {
+    align-items: center;
+    border: 1px solid;
+    box-sizing: border-box;
+    display: flex;
+    transition: all 0.2s ease;
+    width: 100%;
+
+    @media (width <= 600px) {
```

Comment:
remove this approach and use the approach in response_to 
/Users/ditmar/seminario/OpenScience/src/globals/_mixins.scss 
please review that mixins